Content.Server/DeltaV/Xenoarchaeology/XenoArtifacts/Effects/Components/PsionicProducingArtifactComponent.cs

```diff
@@ -0,0 +1,22 @@
+using Content.Server.DeltaV.Xenoarchaeology.XenoArtifacts.Effects.Systems;
+
+namespace Content.Server.DeltaV.Xenoarchaeology.XenoArtifacts.Effects.Components;
+
+/// <summary>
+/// Makes people in a radius around it psionic when triggered.
+/// </summary>
+[RegisterComponent, Access(typeof(PsionicProducingArtifactSystem))]
+public sealed partial class PsionicProducingArtifactComponent : Component
+{
+    /// <summary>
+    /// Range to look for potential psionics in.
+    /// </summary>
+    [DataField(required: true)]
+    public float Range;
+
+    /// <summary>
+    /// Number of times this node can trigger before it switches to doing nothing.
+    /// </summary>
+    [DataField]
+    public int Limit = 1;
```

Comment:
One of my first thoughts was that this was a little too powerful without at least a "_low chance_ of making everyone in X radius a psychic", but this works too!